fixtures/cameo/ts60.json

```diff
@@ -0,0 +1,334 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "ts60",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Anonymous", "Luke Nelson"],
+    "createDate": "2022-10-19",
+    "lastModifyDate": "2022-10-19"
+  },
+  "links": {
+    "productPage": [
+      "https://www.cameolight.com/en/solutions/dj-musicians/static-lighting/theater-lights/11911/ts-60-w-rgbw",
+      "https://www.cameolight.com/en/solutions/dj-musicians/static-lighting/theater-lights/13839/ts-60-w-rgbw-wh"
+    ],
+    "manual": ["https://www.cameolight.com/en/downloads/file/id/1201324186"]
+  },
+  "physical": {
+    "dimensions": [145, 205, 240],
+    "weight": 3.6,
+    "power": 65,
+    "DMXconnector": "3-pin",
+    "lens": {
+      "name": "Plano-convex",
+      "degreesMinMax": [9, 41]
+    }
+  },
+  "availableChannels": {
+    "Dimmer": {
+      "fineChannelAliases": ["Dimmer fine"],
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Strobe": {
+      "capabilities":[
+        {
+          "dmxRange": [0, 5],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [6, 255],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Strobe",
+          "speedStart": "<1Hz",
```

Comment:
Is `<1Hz` an allowed value?

Reply:
Nope. Answered by the [failed build](https://github.com/OpenLightingProject/open-fixture-library/actions/runs/3284897073/jobs/5411391642#step:9:165).